responsibleai/responsibleai/managers/explainer_manager.py

```diff
@@ -97,6 +98,22 @@ def add(self):
             raise UserConfigValidationException(
                 'Model is required for model explanations')
 
+        if len(set(_find_features_having_missing_values(
```

Comment:
nit: `any(...)` is preferable here because we don't actually care about the length, but rather that there are any elements (not zero).

Reply:
done